Update active-directory-aadconnectsync-feature-prevent-accidental-del…

HeathL17 opened the pull request:
…etes.md

Updating to include clearer instruction that the deletion threshold cmdlets are part of the AD Sync module.

deneha commented:
@Erikre Please signoff by leaving a comment in this pull request when you've reviewed your edits, the edits all appear as expected, and you're ready for review prior to publish.  Thanks!

Erikre commented:
This article has been updated, so there's no need for this PR. Closing.

HeathL17 committed:
Update active-directory-aadconnectsync-feature-prevent-accidental-deletes.md

Updating to include clearer instruction that the deleteion threshold cmdlets are part of the AD Sync module.

MicrosoftGuyJFlo reviewed (changes requested):
Line 1 needs to be fixed or will cause an issue when publishing. Please revise back to original ----

HeathL17 committed:
Update active-directory-aadconnectsync-feature-prevent-accidental-deletes.md

MicrosoftGuyJFlo reviewed (approved):
Changes look good. Thank you! Approved.
